NtupleProducer/src/combiner.cc

```diff
@@ -359,7 +364,7 @@ void combiner::computeWeights(){
     if (fParticles[i0].pdgId() == 4) { fParticlesPuppi.push_back(fParticles[i0]); puppictr++; }
     if (fParticles[i0].puppiWeight() <= 0.01) continue;
     if (fParticles[i0].pdgId() != 4 && fParticles[i0].puppiWeight() > 0.01){
-      if (fabs(fParticles[i0].eta() < fEta)){
+      if (fabs(fParticles[i0].eta()) < fEta){
```

Comment:
I believe this was really a bug, we were using the charged alpha in the negative half of the barrel.
@nhanvtran @violatingcp